docs/requirement-sets/word/word-api-desktop-requirement-set.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Word JavaScript API desktop-only requirement set
+description: Details about the WordApiHiddenDocument requirement set.
+ms.date: 06/24/2022
+ms.prod: word
+ms.localizationpriority: medium
+---
+
+# Word JavaScript API desktop-only requirement set
```

Comment:
I waffled on suggesting this page just be called "HiddenDesktop 1.3". I think what you have here better describes the implications for the user. That said, if Word does go down a path of more desktop-only APIs, there won't just be one requirement set (like WordOnlineApi). Online gets away with that because there's one version of the web client. It's entirely possible desktop-only sets will stagger across SAC updates before being rolled into a traditional numbered re set and persist for a while.